### DISCUSSION TITLE
Presentation about clangd on Jan 13

### DISCUSSION THREAD

**Discussion by HighCommander4** [clangd/clangd#984] [Show and tell]
Not sure if this sort of post is appropriate in this forum (if not, please delete), but I thought I'd mention that I'll be giving a [virtual talk](https://strumenta.community/meetup/virtual-meetup-botond-ballo/) about C++ editor tooling, with a primary focus on clangd, on Jan 13.

**Comment by kadircet** [comment]
i am not sure about the medium either, but it's definitely great to hear
about it! i think having such announcements here or on discord is
definitely a good idea. thanks for letting us know!
